manifests/overlays/prod/TEMPLATE/secret.enc.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: solgate-NAME
+stringData:
+    source.creds.yaml: ENC[AES256_GCM,data:aHF12Hcy8Pyz7VrT8BIlYWZMc5D/Jm0l/+7jGKb0+tVBbzQoGvaljbftFg0NKrkzNi40954g,iv:8pmM9rhaCUuLYxFAbZp9PWEhe0YXXvhWekRdX/NnVKE=,tag:0lz6GA3GCNlrsF359gs5dA==,type:str]
+    destination.creds.yaml: ENC[AES256_GCM,data:kL/oTDEB8V1DfUHY+7/1zC812NyiepTm545xglHjk6FAKfSJ+72/msZX8ngeBnHHlVKG2E1c,iv:NSSrgouX0kxRTFatUIOjaNjOm5x5dkGMobbHqDlcnvo=,tag:OL5LOZyJ9hubs6g/RRHD1g==,type:str]
+    config.yaml: |
+        alerts_smtp_server: smtp.corp.redhat.com
+        alerts_from: solgate-alerts@redhat.com
+        alerts_to: data-hub-alerts@redhat.com
```

Comment:
Should we remove this mail list from the default template?

Reply:
Would it be sufficient if I comment it out? :slightly_smiling_face: